src/components/ChannelPopup.svelte

```diff
@@ -37,6 +37,7 @@
             type="button"
             area-selected="{view === 'html'}"
             on:click="{() => switchView('html')}"
+            on:keypress="{() => switchView('html')}"
```

Comment:
This expression means that the function will be triggered by pressing any key on the keyboard and only if this button is already in focus. Are you sure this is what you want?

If you only want to add support for navigation with Tab and Enter, then it already works on all buttons by default.

Reply:
It seems Svelte is asking for it by default, according to A11y standards. Though I indeed had the Tab+Enter navigation in mind.